Rakefile

```
@@ -18,6 +20,14 @@ CLOBBER.add("ports/*").exclude(%r{ports/archives$})
 
 RE2_GEM_SPEC = Gem::Specification.load('re2.gemspec')
 
+task :prepare do
+  puts "Preparing project for gem building..."
+  recipes = load_recipes
+  recipes.each { |recipe| recipe.download }
```

Bit of code golf but...

```suggestion
  load_recipes.each(&:download)
```